File: FirstBit Workspace/C Programming/Assignments/Assignment 9/Q4.c
//Find odd and even among the numbers.
#include<stdio.h>
void arrayInputOutput(int*, int);
void evenOddArray(int*, int);
void main(){
	
	int arr[10];
	
	arrayInputOutput(arr, 10);
	evenOddArray(arr,10);
		
}	

void arrayInputOutput(int* ptr, int size){
	
	//storing numbers in a array
	printf("Enter 10 numbers to store in a array: ");
	for(int i=0;i<size;i++)
		scanf("%d",&ptr[i]);
		
	printf("\nYour array: ");
	for(int i=0;i<size;i++)
		printf("%d ",ptr[i]);
}

void evenOddArray(int* ptr, int size){

	printf("\n\nEven elements: ");
	for(int i=0;i<size;i++)
		if(ptr[i]%2==0)
			printf("\nindex %d: %d ",i,ptr[i]);
	
	printf("\n\nOdd elements: ");
	for(int i=0;i<size;i++)
		if(ptr[i]%2!=0)
			printf("\nindex %d: %d ",i,ptr[i]);
}

File: FirstBit Workspace/Core Java/In Class Work/ArrayCProgramsInJava/src/Q4.java
import java.util.Scanner;

public class Q4 {
	
	public static void main(String[] args) {
		//Find odd and even among the numbers.
		
				//array reference 
				int[] array;
				
				//initialization
				array = new int[5];
				
				Scanner sc = new Scanner(System.in);
				
				System.out.println("Enter elements to store in array: ");
				
				//store elements
				for(int i=0; i<array.length; i++) {
					array[i] = sc.nextInt(); 
				}
				
				System.out.println("\nElements stored in array: ");
				//display array
				for(int ele:array) {
					System.out.println(ele);
				}
				
				System.out.print("\nEven Numbers in array: ");
				
				for(int ele:array) {
					if(ele % 2 == 0) System.out.print(ele + " ");
				}
				
				System.out.print("\nOdd Numbers in array: ");
				
				for(int ele:array) {
					if(ele % 2 != 0) System.out.print(ele + " ");
				}
				
				sc.close();
	}

}
